### PR TITLE
test: do explicit check of CORS headers

### PR DESCRIPTION
**Pull request by Kubuxu**
License: MIT
Signed-off-by: Jakub Sztandera <kubuxu@protonmail.ch>

### CODE REVIEW

File: test/sharness/t0112-gateway-cors.sh [hunk 1]
```diff
@@ -28,10 +28,11 @@ thash='QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn'
 test_expect_success "GET to Gateway succeeds" '
     curl -svX GET "http://127.0.0.1:$gwport/ipfs/$thash" 2>curl_output
 '
+cat curl_output
```
Comment (lgierth) [inline test/sharness/t0112-gateway-cors.sh]: is this a stray debugging line?

Comment (Kubuxu) [inline test/sharness/t0112-gateway-cors.sh]: Yes, thanks.

File: test/sharness/t0112-gateway-cors.sh [hunk 2]
```diff
@@ -28,10 +28,11 @@ thash='QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn'
 test_expect_success "GET to Gateway succeeds" '
     curl -svX GET "http://127.0.0.1:$gwport/ipfs/$thash" 2>curl_output
 '
+
 # GET Response from Gateway should contain CORS headers
 test_expect_success "GET response for Gateway resource looks good" '
-    grep "Access-Control-Allow-Origin:" curl_output &&
-    grep "Access-Control-Allow-Methods:" curl_output &&
+    grep "Access-Control-Allow-Origin:" curl_output | grep "\*"  &&
+    grep "Access-Control-Allow-Methods:" curl_output | grep GET &&
```
Comment (lgierth) [inline test/sharness/t0112-gateway-cors.sh]: This will match GETT :P

**Comment by whyrusleeping** [comment]
Resolves: https://github.com/ipfs/go-ipfs/pull/2778

**Comment by jbenet** [comment]
What does this resolve? can you copy-paste the line or link to it in the review? or link from there to here?

**Comment by jbenet** [comment]
Okay thanks @whyrusleeping

**Comment by jbenet** [comment]
This LGTM. merge it and link to it in the review issue.

**Pull request merged**